src/js/components/Layer/LayerContainer.js

```diff
@@ -127,8 +144,8 @@ const LayerContainer = forwardRef(
           dir={theme.dir}
         >
           <StyledOverlay
+            ref={overlayRef}
             plain={plain}
-            onMouseDown={onClickOutside}
```

Comment:
Can we just leave this as `onMouseDown={onClickOutsize}`? I'm not sure what adding the `useEffect()` is doing beyond this. Maybe I'm missing something?